hackathon_site/requirements.txt

```diff
@@ -10,6 +10,7 @@ dj-rest-auth==1.0.6
 Django==3.1.8
 django-cors-headers==3.3.0
 django-debug-toolbar==2.2.1
+django-filter==2.4.0
```

Comment:
I think these changes were already merged into develop in a separate PR? 

Looking at the git log, it looks like you branched this branch off of your local ieee-2 branch. That's generally not a good idea, since it will end up making changes appear again in the PR (the technical reason being that we squash and merge PRs which creates a new commit, so the old one you branched off isn't recognized).

Technically the way to fix this is to rebase, but I think pulling and merging develop in will fix this (if not, we can delete the offending commit from this branch and force push). In the future, make sure you always branch off develop.